File: BungiiAutomation/Bungii.Test.Regression.Web.Integration/Pages/Admin/Admin_MenuLinksPage.cs
using OpenQA.Selenium;
using OpenQA.Selenium.Support.PageObjects;

namespace Bungii.Android.Regression.Test.Integration.Pages.Admin
{
    public class Admin_MenuLinksPage
    {
        public Admin_MenuLinksPage(IWebDriver webdriver)
        {
            PageFactory.InitElements(webdriver, this);
        }

        [FindsBy(How = How.Id, Using = "adminmenu-dashboard")]
        public IWebElement Menu_Dashboard { get; set; }

        [FindsBy(How = How.Id, Using = "adminmenu-customers")]
        public IWebElement Menu_Customers { get; set; }

        [FindsBy(How = How.Id, Using = "adminmenu-drivers")]
        public IWebElement Menu_Drivers { get; set; }

        [FindsBy(How = How.Id, Using = "adminmenu-trips")]
        public IWebElement Menu_Trips { get; set; }

        [FindsBy(How = How.Id, Using = "adminmenu-completedtrips")]
        public IWebElement Menu_Trips_Trips { get; set; }

        [FindsBy(How = How.Id, Using = "adminmenu-livetrips")]
        public IWebElement Menu_Trips_LiveTrips { get; set; }

        [FindsBy(How = How.Id, Using = "adminmenu-marketing")]
        public IWebElement Menu_Marketing { get; set; }

        [FindsBy(How = How.Id, Using = "adminmenu-promocode")]
        public IWebElement Menu_Marketing_PromoCode { get; set; }

        [FindsBy(How = How.Id, Using = "adminmenu-referralsource")]
        public IWebElement Menu_Marketing_ReferralSource { get; set; }

        [FindsBy(How = How.Id, Using = "adminmenu-geofences")]
        public IWebElement Menu_Geofences { get; set; }
    }
}

File: Bungii.Test.Integration/src/test/java/com/bungii/web/pages/admin/Admin_MenuLinksPage.java
package com.bungii.web.pages.admin;

import com.bungii.common.core.PageBase;
import org.openqa.selenium.WebElement;

public class Admin_MenuLinksPage extends PageBase {

    public WebElement Menu_Dashboard () { return findElement("adminmenu-dashboard", LocatorType.Id); }

    public WebElement Menu_Customers () { return findElement("adminmenu-customers", LocatorType.Id); }

    public WebElement Menu_Drivers () { return findElement("adminmenu-drivers", LocatorType.Id); }

    public WebElement Menu_Trips () { return findElement("adminmenu-trips", LocatorType.Id); }

    public WebElement Menu_Trips_Trips () { return findElement("adminmenu-completedtrips", LocatorType.Id); }

    public WebElement Menu_Trips_LiveTrips () { return findElement("adminmenu-livetrips", LocatorType.Id); }

    public WebElement Menu_Marketing () { return findElement("adminmenu-marketing", LocatorType.Id); }

    public WebElement Menu_Marketing_PromoCode () { return findElement("adminmenu-promocode", LocatorType.Id); }

    public WebElement Menu_Marketing_ReferralSource () { return findElement("adminmenu-referralsource", LocatorType.Id); }

    public WebElement Menu_Geofences () { return findElement("adminmenu-geofences", LocatorType.Id); }
}
